extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -21,125 +21,110 @@
-// running in.  In this experiment phase, we're testing 8 different
-// configurations, resulting from the Cartesian product of the following:
-//   - Traditional 3p iframe ad rendering (control) vs A4A rendering
-//     (experiment)
-//   - Experiment triggered by an external page, such as the Google Search
-//     page vs. triggered internally in the client code.
-//   - Doubleclick vs Adsense
-// The following two objects contain experiment IDs for the first two
-// categories for Doubleclick ads.  They are attached to the ad request by
-// ads/google/a4a/traffic-experiments.js#googleAdsIsA4AEnabled when it works
-// out whether a given ad request is in the overall experiment and, if so,
-// which branch it's on.
-
-// We would prefer the following constants to remain private, but we need to
-// refer to them directly in amp-ad-3p-impl.js and amp-a4a.js in order to check
-// whether we're in the experiment or not, for the purposes of enabling
-// debug traffic profiling.  Once we have debugged the a4a implementation and
-// can disable profiling again, we can return these constants to being
-// private to this file.
-/** @const {!../../../ads/google/a4a/traffic-experiments.A4aExperimentBranches} */
-export const DOUBLECLICK_A4A_EXTERNAL_EXPERIMENT_BRANCHES_PRE_LAUNCH = {
-  control: '117152662',
-  experiment: '117152663',
-};
+export const DOUBLECLICK_A4A_EXPERIMENT_NAME = 'expDoubleclickA4A';
 
-export const DOUBLECLICK_A4A_EXTERNAL_DELAYED_EXPERIMENT_BRANCHES_PRE_LAUNCH = {
-  control: '117152664',
-  experiment: '117152665',
-};
-
-/**
- * @const {!../../../ads/google/a4a/traffic-experiments.A4aExperimentBranches}
- */
-export const DOUBLECLICK_A4A_EXTERNAL_EXPERIMENT_BRANCHES_POST_LAUNCH = {
-  control: '2092619',
-  experiment: '2092620',
-};
-
-/**
- * @const {!../../../ads/google/a4a/traffic-experiments.A4aExperimentBranches}
- */
-export const DOUBLECLICK_A4A_INTERNAL_EXPERIMENT_BRANCHES_PRE_LAUNCH = {
-  control: '117152680',
-  experiment: '117152681',
-};
-
-/**
- * @const {!../../../ads/google/a4a/traffic-experiments.A4aExperimentBranches}
- */
-export const DOUBLECLICK_A4A_INTERNAL_EXPERIMENT_BRANCHES_POST_LAUNCH = {
-  control: '2092613',
-  experiment: '2092614',
-};
+/** @type {string} */
+const TAG = 'amp-ad-network-doubleclick-impl';
 
-/**
- * @const {!../../../ads/google/a4a/traffic-experiments.A4aExperimentBranches}
- */
-export const DOUBLECLICK_A4A_BETA_BRANCHES = {
-  control: '2077830',
-  experiment: '2077831',
+/** @const @enum{string} */
+export const DOUBLECLICK_EXPERIMENT_FEATURE = {
+  HOLDBACK_EXTERNAL: '2092620',
+  HOLDBACK_INTERNAL: '2092614',
+  DELAYED_REQUEST: '117152665',
+  SRA: '117152667',
 };
 
-/**
- * @const {!../../../ads/google/a4a/traffic-experiments.A4aExperimentBranches}
- */
-export const DOUBLECLICK_SFG_INTERNAL_EXPERIMENT_BRANCHES = {
-  control: '21060540',
-  experiment: '21060541',
+/** @const @type {!Object<string,?string>} */
+export const URL_EXPERIMENT_MAPPING = {
```

Comment:
could we maybe have all these values within this object as consts at the top to make the mapping more readable? The mix of strings and consts in here is not the prettiest